<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBackGround</class>
 <widget class="QMainWindow" name="MainBackGround">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/icons8-stopwatch-66.png</normaloff>../Icons/icons8-stopwatch-66.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="Container">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="CircleBG">
         <property name="styleSheet">
          <string notr="true">QFrame{
	
	background-color: rgb(255,255, 255, 15);
     
	
    border-radius:245px;
}
&quot;background-color:rgba(255,255, 25, 150);&quot;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="StopBTN">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>370</y>
            <width>52</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton
{
    color: #FFFFFF;
    background-color:none;
    border-width: 1px;
    border-color: #FFFFFF;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}
QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
	
}

QComboBox:hover,QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="ResetBTN">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>420</y>
            <width>52</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton
{
    color: #FFFFFF;
    background-color:none;
    border-width: 1px;
    border-color: #FFFFFF;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}
QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="IconORgif">
          <property name="geometry">
           <rect>
            <x>226</x>
            <y>375</y>
            <width>40</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../Icons/stopwatch.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="StartBTN">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>370</y>
            <width>52</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton
{
    color: #FFFFFF;
    background-color:none;
    border-width: 1px;
    border-color: #FFFFFF;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    	
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_Sep1">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>160</y>
            <width>27</width>
            <height>151</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:none;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_Sep2">
          <property name="geometry">
           <rect>
            <x>314</x>
            <y>160</y>
            <width>27</width>
            <height>151</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:none;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_MS">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>122</width>
            <height>106</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>48</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:none;
font: 48pt &quot;Segoe UI&quot;;
font-weight:600;
color: rgb(170, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; color:#aaffff;&quot;&gt;MS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_AppName">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>110</y>
            <width>323</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>22</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:none;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;StopWatch &lt;span style=&quot; font-weight:600; color:#de18f0;&quot;&gt;&amp;amp; &lt;/span&gt;Timer&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
         <widget class="QPushButton" name="CloseBTN">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>60</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: transparent;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/CloseIcon.png</normaloff>../Icons/CloseIcon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="QLineEdit" name="Seconds">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>194</y>
            <width>151</width>
            <height>111</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>72</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(255, 255, 255,255);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
          <property name="text">
           <string>00</string>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="Minutes">
          <property name="geometry">
           <rect>
            <x>175</x>
            <y>194</y>
            <width>151</width>
            <height>111</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>72</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(255, 255, 255,255);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>00</string>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="Hours">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>194</y>
            <width>151</width>
            <height>111</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>72</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(255, 255, 255,255);
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>00</string>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Timer">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>327</y>
            <width>220</width>
            <height>30</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
    color: #FFFFFF;
    background-color:none;
    border-width: 1px;
    border-color: #FFFFFF;
    border-style: solid;
    border-bottom-left-radius :30px;
    border-bottom-right-radius : 15px;
    border-top-right-radius :15px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}
QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
 
}

QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QPushButton:checked
{ 
   
	
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);

 }</string>
          </property>
          <property name="text">
           <string>Timer</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="StopWatch">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>327</y>
            <width>220</width>
            <height>30</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
    color: #ffffff;
    background-color:none;
    border-width: 1px;
    border-color: #FFFFFF;
    border-style: solid;
    border-bottom-left-radius : 15px;
    border-top-left-radius :15px;
    border-bottom-right-radius : 30px;

    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
    min-width: 40px;
}
QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
 
}

QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}

QPushButton:checked
{ 
   
	
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);

 }</string>
          </property>
          <property name="text">
           <string>StopWatch</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="autoRepeatDelay">
           <number>300</number>
          </property>
          <property name="autoRepeatInterval">
           <number>100</number>
          </property>
         </widget>
         <widget class="QPushButton" name="Folder">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: transparent;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/FolderIcon.png</normaloff>../Icons/FolderIcon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="Back">
          <property name="geometry">
           <rect>
            <x>189</x>
            <y>170</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: transparent;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/BackIcon.png</normaloff>../Icons/BackIcon.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="PlayBtn">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>163</y>
            <width>35</width>
            <height>35</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: transparent;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/playIcon.png</normaloff>../Icons/playIcon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="Next">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>170</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color: transparent;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
    
}
QPushButton:hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/NextIcon.png</normaloff>../Icons/NextIcon.png</iconset>
          </property>
         </widget>
         <zorder>StopBTN</zorder>
         <zorder>ResetBTN</zorder>
         <zorder>StartBTN</zorder>
         <zorder>label_Sep1</zorder>
         <zorder>label_Sep2</zorder>
         <zorder>label_MS</zorder>
         <zorder>label_AppName</zorder>
         <zorder>CloseBTN</zorder>
         <zorder>IconORgif</zorder>
         <zorder>Seconds</zorder>
         <zorder>Minutes</zorder>
         <zorder>Hours</zorder>
         <zorder>Timer</zorder>
         <zorder>StopWatch</zorder>
         <zorder>Folder</zorder>
         <zorder>Back</zorder>
         <zorder>PlayBtn</zorder>
         <zorder>Next</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
